lib/index.js

```diff
@@ -1,5 +1,13 @@
 var xtend = require('xtend');
 
+var ERROR_CORS_REQUEST              = 'Can not read cookies from CORS-Requests. See CORS-Workaround in the readme.';
+var ERROR_NO_COOKIE_PARSER          = 'cookieParser is required use require(\'cookie-parser\'), connect.cookieParser or express.cookieParser';
+var ERROR_NO_SESSION                = 'No session found';
+var ERROR_PASSPORT_NOT_INITIALIZED  = 'Passport was not initialized';
+var ERROR_SESSION_STORE             = 'Error in session store:\n';
```

Comment:
The `\n` should probably not be in the constant.. I'll move it to the message concatenaed to it on line 79.